app/src/main/res/values/dimens.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
-    <dimen name="fab_margin">16dp</dimen>
+    <dimen name="actors_margin">8dp</dimen>
+    <dimen name="views_margin_16">16dp</dimen>
+    <dimen name="views_margin_8">8dp</dimen>
+    <dimen name="text_storyline_padding">4dp</dimen>
+    <dimen name="text_description_padding">20dp</dimen>
+    <dimen name="text_cast_padding">8dp</dimen>
```

Comment:
Add dimens

Reply:
You already have a couple of 8dp duplications. If you go like this the file will grow enormously.